cppcache/src/TcrMessage.hpp

```diff
@@ -675,9 +493,7 @@ class TcrMessageClearRegion : public TcrMessage {
                         std::chrono::milliseconds messageResponsetimeout,
                         ThinClientBaseDM* connectionDM);
 
-  virtual ~TcrMessageClearRegion() {}
-
- private:
+  ~TcrMessageClearRegion() override;
```

Comment:
Why not `~TcrMessageClearRegion() noexcept override = default;`?

At least do `noexcept` on these cleanups.

Reply:
I was trying not to get too fancy about changing implementation details - the idea was really just to move stuff without getting carried away. We shouldn't have to do ```noexcept```, as that's implicit since C++11, which is our target spec. I do know default doesn't always generate the right thing, I need to read up on the subject before I commit to changing the code accordingly. Technically I screwed up with override, which I should remove.

Reply:
C++11 noexcept is implicitly true. I defaulted the derived destructors.